tests/legacy/ApiTest.php

```diff
@@ -691,26 +692,25 @@ public function testApiAccountUpdateProfileImageWithUpload()
 	}
 
 	/**
-	 * Test the check_acl_input() function.
+	 * Test the ACL::isValidContact() function.
 	 *
 	 * @return void
 	 */
 	public function testCheckAclInput()
```

Comment:
It doesn't seem as if this test already exists. I'm unsure how to set it up.

Reply:
Create a new class with 
```php
class ACLTest extends FixtureTest { }
```
And move the method inside this class. I think this should work.

I don't want to commit the test for you yet, because as far as I can see, you're currently working as well at your branch and I don't want to compromise it :-)